cpp/src/io/orc/stripe_enc.cu

```diff
@@ -49,7 +49,7 @@ constexpr int scratch_buffer_size = 512 * 4;
 // Workaround replaces zero-length patch lists by a dummy zero patch
 constexpr bool zero_pll_war = true;
 
-static __device__ __constant__ int64_t kORCTimeToUTC =
+static __device__ __constant__ int64_t orc_utc_epoch =
```

Comment:
This constant looks similar to the `duration_s` in `stripe_data.cu`. I'm not sure I've understood why we have two constants for the same value.

Reply:
it is the same constant, just for the reader and the writer. I'll try to move it to a common header.